appmanifest/id-member/id-member-manual-tentative-service-worker.js

```diff
@@ -0,0 +1,61 @@
+// Some user agents only offer app installation if there is a SW and it handles
+// offline requests.
+
+const cacheVersion = "1.1";
+const CACHE_NAME = `cache-v${cacheVersion}`;
+
+// The resources cached by this service worker.
```

Comment:
Is all of this caching recommended by WPT tests? Is there a reason why you are doing it?

If not, the test is simpler if we don't do it - I think we might have an offline requirement for installability at some point? So this serviceworker would work:


```js
self.addEventListener('fetch', event => {
  event.respondWith(fetch(event.request).catch(_ => {
    return new Response('Offline test.');
  }));
});
```